newrelic_lambda_cli/integrations.py

```diff
@@ -85,6 +85,54 @@ def _get_cf_stack_status(session, stack_name, nr_account_id=None):
     else:
         return res["Stacks"][0]["StackStatus"]
 
+def get_unique_newrelic_log_ingestion_name(session):
+    stack_id = _get_cf_stack_id(session, stack_name=INGEST_STACK_NAME)
+    if stack_id:
+        return "newrelic-log-ingestion-%s"%(stack_id.split("/")[2].split("-")[4])
+    return "newrelic-log-ingestion"
```

Comment:
Just return None (which is the default behavior if you don't specify a return at all) if the stack does not exist.
```suggestion
```

Reply:
The function needs to return string for the nested stack in the update_log_ingestion_function which in turn calls [import_log_ingestion_function](https://github.com/newrelic/newrelic-lambda-cli/pull/254/files#:~:text=Starting%20import%22)